reference/resources/felixconfig.md

```diff
@@ -39,6 +39,7 @@ spec:
 
 | Field                              | Description                 | Accepted Values   | Schema | Default    |
 |------------------------------------|-----------------------------|-------------------|--------|------------|
+| awsSrcDstCheck                     | Controls automatically setting {% include open-new-window.html text='source-destination-check' url='https://docs.aws.amazon.com/vpc/latest/userguide/VPC_NAT_Instance.html#EIP_Disable_SrcDestCheck' %} on an AWS EC2 instance running Felix. Setting the value to `Enable` will set the check value in the instance description to `true`. For `Disable`, the check value will be `false`. Enabling this setting is required if the EC2 instance can process traffic not matching interface IP address, e.g. EKS cluster using Calico CNI with `VXLANMode=CrossSubnet`. | DoNothing, Enable, Disable | string | `DoNothing` |
```

Comment:
This sentence is confusing because of the "can process and "not matching" clauses together. 

"Set this if the EC2 instance can process traffic not matching interface IP address, e.g. EKS cluster using Calico CNI with `VXLANMode=CrossSubnet`"

Is this correct? 

Setting must be `Enable` if the EKS cluster is using Calico CNI with `VXLANMode=CrossSubnet`.